api/src/main/java/cz/muni/fi/pa165/facade/LaboratoryFacade.java

```diff
@@ -0,0 +1,27 @@
+package cz.muni.fi.pa165.facade;
+
+import cz.muni.fi.pa165.dto.BottleDTO;
+import cz.muni.fi.pa165.dto.LaboratoryDTO;
+import cz.muni.fi.pa165.dto.PersonDTO;
+import java.util.List;
+
+/**
+ *
+ * @author rk
+ * @date 2016-11-22
+ */
+public interface LaboratoryFacade {
+    void createLaboratory(LaboratoryDTO laboratory);
+    
+    List<LaboratoryDTO> findAll();
+
+    LaboratoryDTO findById(Long id);
+    
+    LaboratoryDTO findByName(String name);
+    
+    List<PersonDTO> findAllPersonsInLaboratory();
```

Comment:
Nechyba ti tu parameter, ze v ktorom laboratory hladas tych personov? A ked uz budes mat to laboratory, tak ti staci na tom zavolat getPersons()